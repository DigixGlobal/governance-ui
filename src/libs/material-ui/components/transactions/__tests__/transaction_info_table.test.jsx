import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TransactionInfoTable from '../transaction_info_table';

jest.mock('../../../helpers/stringUtils', () => ({
  parseBigNumber: () => 20,
}));

const txData = {
  tx1: 10,
  tx2: 20,
  tx3: 30,
};

describe('<TransactionInfoTable />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TransactionInfoTable txData={txData} />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
