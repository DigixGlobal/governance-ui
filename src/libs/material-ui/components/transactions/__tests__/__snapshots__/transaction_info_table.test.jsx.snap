// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionInfoTable /> renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <Button
      as="button"
      basic={true}
      content="Details"
      icon="angle double down"
      onClick={[Function]}
      size="mini"
    />
  </div>
</div>
`;
